src/grafana_dashboards/metrics.json

```diff
@@ -1575,8 +1577,8 @@
       "type": "row"
     }
   ],
-  "refresh": false,
-  "schemaVersion": 37,
+  "refresh": "",
```

Comment:
Is this intended?
The value was `false` and now `""`, which are different data types.

Reply:
I exported the json directly from Grafana, it seems that setting false or empty string has the same intention (no live refresh): https://github.com/grafana/grafonnet-lib/issues/45 . Maybe when this panel was first created some time ago, a different version of Grafana was used for exporting, which used false by default. I do not expect this code change to have any effect.